content/blog/2020/descriptors_in_the_wild.md

```diff
@@ -42,10 +42,10 @@ transaction.
 ## The role of Descriptors
 
 If Alice and Bob cannot agree on the software to use, to monitor the same financial 
-situation, the two software must control and produce exactly the same series
+situation, the two softwares must control and produce exactly the same series
```

Comment:
sorry plural of software is software.. English grammar is weird. 
https://thewordcounter.com/plural-of-software/

Reply:
So I guess I won't correct it. I will make a PR in your repository so you can apply a new PR.